### PR TITLE
K8SPXC-895 - Update for PXC Operator release 1.10.0

### PR DESCRIPTION
**Pull request by tplavcic**
This merges community contributions from PR's: #99, #100 and #101

### CODE REVIEW

File: charts/pxc-db/templates/cluster.yaml [hunk 1]
```diff
@@ -38,9 +38,13 @@ spec:
   {{- else }}
   logCollectorSecretName: {{ include "pxc-database.fullname" . }}-log-collector
   {{- end }}
+  {{- if .Values.initImage }}
+  initImage: {{ .Values.initImage }}
```
Comment (cap1984) [inline charts/pxc-db/templates/cluster.yaml:42]: is operator able to handle empty `initImage` field without crash or error?

File: charts/pxc-db/templates/cluster.yaml [hunk 2]
```diff
@@ -49,25 +53,44 @@ spec:
...
   pxc:
     size: {{ $pxc.size }}
     image: "{{ $pxc.image.repository }}:{{ $pxc.image.tag }}"
     autoRecovery: {{ $pxc.autoRecovery }}
+    {{- if $pxc.schedulerName }}
+    schedulerName: {{ $pxc.schedulerName }}
+    {{- end }}
     readinessDelaySec: {{ $pxc.readinessDelaySec }}
     livenessDelaySec: {{ $pxc.livenessDelaySec }}
-    forceUnsafeBootstrap: {{ $pxc.forceUnsafeBootstrap }}
```
Comment (cap1984) [inline charts/pxc-db/templates/cluster.yaml:59]: removed since obsolete?

File: charts/pxc-db/templates/cluster.yaml [hunk 3]
```diff
@@ -311,12 +403,12 @@ spec:
 {{ tpl ($pmm.resources.limits | toYaml) $ | indent 8 }}
   {{- end }}
   {{- $backup := .Values.backup }}
-
+  {{- if $backup.enabled }}
```
Comment (cap1984) [inline charts/pxc-db/templates/cluster.yaml:406]: Removing the whole backup section if enabled not specified?

File: charts/pxc-db/templates/cluster.yaml (hunk 1 above)
Comment (tplavcic) [reply charts/pxc-db/templates/cluster.yaml:42]: Yes, by default it uses same as operator image, this is specified only if some custom initImage needed.

File: charts/pxc-db/templates/cluster.yaml (hunk 2 above)
Comment (tplavcic) [reply charts/pxc-db/templates/cluster.yaml:59]: Yes, removed under this ticket: https://jira.percona.com/browse/K8SPXC-831

File: charts/pxc-db/templates/cluster.yaml (hunk 1 above)
Comment (tplavcic) [reply charts/pxc-db/templates/cluster.yaml:41]: I have simplified it now and removed the `if`.

**Review by cap1984 — APPROVED** [review]

**Review by cap1984 — APPROVED** [review]

**Pull request merged**